src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -1436,6 +1436,23 @@ class MavlinkStreamCameraTrigger : public MavlinkStream
 				msg_cmd.param7 = NAN;
 
 				mavlink_msg_command_long_send_struct(_mavlink->get_channel(), &msg_cmd);
+
+				/* send MAV_CMD_DO_DIGICAM_CONTROL*/
+				mavlink_command_long_t digicam_ctrl_cmd;
+
+				digicam_ctrl_cmd.target_system = mavlink_system.sysid;
```

Comment:
Thanks @zehortigoza I wasn't aware you could use 0 for broadcast. I'll make the change.